main.go

```diff
@@ -61,6 +62,7 @@ func main() {
 	f.BoolP("version", "v", false, "show version")
 	f.VarP(&diff.valueFiles, "values", "f", "specify values in a YAML file (can specify multiple)")
 	f.StringArrayVar(&diff.values, "set", []string{}, "set values on the command line (can specify multiple or separate values with commas: key1=val1,key2=val2)")
+	f.BoolVar(&diff.reuseValues, "reuse-values", false, "when upgrading, reuse the last release's values, and merge in any new values. If '--reset-values' is specified, this is ignored.")
```

Comment:
I think the second sentence is was copy&pasted from helm and is a little misleading as helm-diff does not have a `--reset-values` flag.

Reply:
You're totally right 😄 , will fix

Reply:
Updated; I just shortened the description to remove the irrelevant bits.